pkg/orchestrator/transformer/job.go

```diff
@@ -38,10 +37,9 @@ func DefaultsApplier(defaults JobDefaults) JobTransformer {
 }
 
 // RequesterInfo is a transformer that sets the requester ID and public key in the job meta.
```

Comment:
Comment now inaccurate as the public key is removed

Reply:
fixed in https://github.com/bacalhau-project/bacalhau/pull/3188/commits/a3132f0fc047f08c078753c4fc23c85f2be940ea